### ISSUE TITLE
FilePackage.java:113-115: Create unit tests to verify...

### ISSUE THREAD

**Issue by 0pdd** [artipie/rpm-adapter#75]
The puzzle `69-3c6bbf6c` from #69 has to be resolved: 

https://github.com/artipie/rpm-adapter/blob/493153a41dfc9dcae2bb43d8af290adf7d7acf47/src/main/java/com/artipie/rpm/pkg/FilePackage.java#L113-L115

The puzzle was created by @g4s8 on 08-Apr-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
The job [#75](https://github.com/artipie/rpm-adapter/issues/75) assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/06b7f013-510e-4529-aa6e-00febfe89ab9); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/75#issuecomment-613764376))

@carlosmiranda The user @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) resigned from [#75](https://github.com/artipie/rpm-adapter/issues/75), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
The job [#75](https://github.com/artipie/rpm-adapter/issues/75) assigned to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/11ee32e3-6f2a-4a8c-b791-8bd40f6a3f70); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by ammaratef45** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/75#issuecomment-619232856))

@ammaratef45 The user @ammaratef45[/z](https://www.0crat.com/u/ammaratef45) resigned from [#75](https://github.com/artipie/rpm-adapter/issues/75), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/75#issuecomment-619232856))

@ammaratef45 Job refused in 19 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
The job [#75](https://github.com/artipie/rpm-adapter/issues/75) assigned to @hdouss[/z](https://www.0crat.com/u/hdouss), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/1ce42361-931c-4acd-a208-10de4a091748); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by HDouss** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/75#issuecomment-619266698))

@HDouss The user @hdouss[/z](https://www.0crat.com/u/hdouss) resigned from [#75](https://github.com/artipie/rpm-adapter/issues/75), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/artipie/rpm-adapter/issues/75#issuecomment-619266698))

@HDouss Job refused in 0 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
The job [#75](https://github.com/artipie/rpm-adapter/issues/75) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/0aee2b6f-aa80-49fd-9f0a-fd6ba9daadfe); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by victornoel** [comment]
@g4s8 I have two questions on this task:
- what is the meaning of testing size or checksum when it corresponds to copypasting the content of the checksum and size method in a test file? Aren't there more useful things to do? Maybe testing directly FilePackage would make more sense, no?
- what is header range? I don't see anything related to that in the class or interface behaviour

**Comment by g4s8** [comment]
@victornoel yes, your suggestions makes sense. This ticket is about verifying metadata parsing from RPM packages and increase coverage for that, since this part is one of the most central point in the logic.

**Comment by victornoel** [comment]
@0crat wait for #152 to be merged

**Comment by victornoel** [comment]
@g4s8 see #152, I added a todo there to convey this intention :)

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#154](https://github.com/artipie/rpm-adapter/issues/154), [#155](https://github.com/artipie/rpm-adapter/issues/155) are still not solved.

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0pdd** [comment]
@0pdd the puzzle [#154](https://github.com/artipie/rpm-adapter/issues/154) is still not solved; solved: [#155](https://github.com/artipie/rpm-adapter/issues/155).

**Comment by 0pdd** [comment]
@0pdd all 2 puzzles are solved here: [#154](https://github.com/artipie/rpm-adapter/issues/154), [#155](https://github.com/artipie/rpm-adapter/issues/155).
